src/Client.php

```diff
@@ -161,6 +174,10 @@ public function getClient()
      */
     protected function restCall($apiUrl, $verb, $guzzleParams, $apiversion='',$headerParams=null)
     {
+        // Populate the log object with request details
+        $logModel = new LogInformation();
+        $logModel-> populateRequestInfo($verb, $apiUrl, $guzzleParams, $this->includeReqResInLogging);
```

Comment:
nit: Would recommend putting includeReqResInLogging into the constructor to the log information class, that way the caller doesnt need to know that they have to call populateRequestInfo for populateResponseInfo to work

Reply:
Well this is how it is designed to work.
The client would just pass the `includeReqResInLogging` into AvaTaxClient constructor and that would initialize this property. An example is like:
`// Create a new client
$client = new Avalara\AvaTaxClient('phpTestApp', '1.0', 'localhost', 'sandbox',[], $logger, true);`

Later the client would only invoke the method to call (using restCall method) and they don't need to worry about passing this value anywhere else because the AvaTaxClient object already has this information.
Let me know if this clarifies.